lib/local_daily_digest.js

```diff
@@ -0,0 +1,448 @@
+var async = require('async'),
+	_ = require('lodash'),
+	log = require('../log'),
+	Profile = require('../models').Profile,
+	Contact = require('../models').Contact,
+	log = require('../log'),
+	mail = require('../mail');
+
+
+var	finalOutput = [];
+
+function get() {
+	var result = {},
+	 	allLocalProfilesWDigest = [],
+	 	tempCheckedOutUsers = [],
+		requiredLocationsId =[],
+	 	requiredUsers=[],
+	 	checkedOutUsers =[],
+		checkedInUsers = [];
+	
+	async.series([
+		function (cb) {
+			//Get a list all of profiles who have the daily digest array and are verified
+			Profile.find( {'dailyDigest': {$not: {$size: 0}}, 'verified': true }, function(err, profile){
```

Comment:
Just for clarity reasons, could you rename profile to profiles (I'm being picky, I know :-) )